src/App.jsx

```diff
@@ -1,3 +1,6 @@
+import { MainPage } from "./components/MainPage";
```

Comment:
Great work! And amazing how much you have thought about accessibility with contrasts and bigger texts.

Reply:
Agree - love the big font size, makes the whole page pop somehow?